test/TestUtilities/Test.Utility/Test.Utility.csproj

```diff
@@ -85,6 +85,7 @@
     <PackageReference Include="FluentAssertions" />
     <PackageReference Include="Microsoft.Build" ExcludeAssets="runtime" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" ExcludeAssets="runtime" />
+    <PackageReference Include="System.Security.Cryptography.Xml" ExcludeAssets="runtime" />
```

Comment:
sort packages, please.

Also, since this doesn't use `PrivateAssets="all"`, then any test project with a project reference to `Test.Utility` will end up with this package transitively. So, I'm not sure all the `PrivateAssets="all"` on all the other test projects makes a difference. Well, maybe it "reverts" the ExcludeAssets in this PackageReference. Note also that only MSBuild packages here have ExcludeAssets, the other packages don't, so I'm not sure that it's needed for this package. If it is, it's not clear why.